kitops/modelkit/manager.py

```diff
@@ -258,7 +258,8 @@ def pack_and_push_modelkit(
         Returns:
             None
         """
-        if save_kitfile:
+        
+        if save_kitfile and not self.kitfile:
```

Comment:
Shouldn't this be `if save_kitfile and self.kitfile`?

Reply:
ugh